app/views/users/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row">
+  <div class="col-md-3"></div>
+  <div class="col-md-6">
+    <% if @user %>
```

Comment:
Is there a reason this line checks that `@user` exists, but the `<%= render @user.posts %>` doesn't check?